extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -521,8 +521,13 @@ export class AmpConsent extends AMP.BaseElement {
 
     this.consentConfig_ = consents;
     if (config['postPromptUI']) {
-      this.postPromptUI_ = this.getAmpDoc().getElementById(
-          config['postPromptUI']);
+      const postPromptUI = config['postPromptUI'];
```

Comment:
Can we allow post-prompt ui to be upscoped, is there a strong reason for scoping it?   I have live sites that this will break.

Reply:
I'm worrying about this
```
<amp-consent>
 postPromptUI = 'test'
</amp-consent>

<div> Content </div>
<div id='test'> postPromptUI </div>
<div> Content </div>
```

By show/hide postPromptUI here, it will introduce a page jump.